src/node_watchdog.h

```diff
@@ -91,6 +91,7 @@ class SigintWatchdogHelper {
   static void* RunSigintWatchdog(void* arg);
   static void HandleSignal(int signum);
 #else
+  bool watchdog_disabled_;
```

Comment:
Tbh, I would find this a bit easier to read as `watchdog_enabled_` with the logic flipped around